docs/adr/0003-remove-jquery-dependency.md

```diff
@@ -0,0 +1,20 @@
+# Remove jQuery and jQuery UI dependencies
+📆 **Updated**: July 7, 2022
+
+🙋🏽‍♀️ **Status** Proposed
+
+## ℹ️ Context
+- The current use of jQuery 1.7.1 is a low vulnerability (XSS)
+- There is not much use of jQuery and jQuery UI, mainly dialogs and tabs
+- It's currently responsible for the 80%+ (LOC) in `site.js`
+
+
+## 🤔 Decision
+Remove jQuery and jQuery UI dependencies. Dialogs and tabs can be implemented in CSS with a sprinkle JavaScript.
+
+## 🎬 Consequences
+- Remove XSS vulnerability
+- Lighter page
+
```

Comment:
Is the full list of dependencies _just_ dialogs and tabs right now?

Reply:
I think there's some stuff in www/site.js
I saw a `$.ajax`

Reply:
Here's a list of what I see in there:
- drag and resizing (not sure where this is used)
- viewport offsets
- getting OS scrollbar width
- displaying local time
- tooltips
- scroll handling 
- editting test labels